src/ORM/FieldType/DBDate.php

```diff
@@ -42,13 +44,14 @@ class DBDate extends DBField
      */
     public const ISO_LOCALE = 'en_US';
 
+    private static array $field_validators = [
+        DateValidator::class,
+    ];
+
     public function setValue(mixed $value, null|array|ModelData $record = null, bool $markChanged = true): static
     {
-        $value = $this->parseDate($value);
```

Comment:
This was essentially validation on set, rather than the standard validate before data write

Reply:
I've reverted this class to how it was in CMS 5 and have made the minimal changes required to support this PR e.g. adding field_validators config, updating method signatures

Reply:
> I've reverted this class to how it was in CMS 5

I assume you mean "to how it was before this PR" - there have been changes to this class in the `6` class that aren't in CMS 5.

Reply:
Yes sorry, I meant CMS 6